client/src/modules/list/index.jsx

```diff
@@ -60,12 +61,14 @@ class List extends Component {
   componentDidMount() {
     this.setState({ pendingForDetails: true });
 
-    this.fetchData().finally(() => {
-      this.setState({ pendingForDetails: false });
-      this.handleBreadcrumbs();
-      this.handleSocketConnection();
-      this.receiveWSEvents();
-    });
+    this.fetchData()
+      .then(() => {
+        this.handleBreadcrumbs();
+        this.handleSocketConnection();
+        this.receiveWSEvents();
+        this.setState({ pendingForDetails: false });
+      })
+      .catch(() => noOp());
```

Comment:
what if an error occurs and this won't be NotFoundExceptio e.g with status 400? You won't be redirected to 404 so how pending will be turning off then? Will this situation cause some problems?